docs/products/kafka/concepts/kafka-tiered-storage.md

```diff
@@ -27,58 +23,87 @@ control over data storage needs.
-    applications and workflows, even those where Apache Kafka might have
-    been considered impractical before. We encourage users to leverage
-    this newfound flexibility to think creatively and redefine their
-    experience with Apache Kafka.
-
-## Pricing
-
-Tiered storage costs are determined by the amount of remote storage
-used, measured in GB/hour. The highest usage level within each hour is
-the basis for calculating charges.
+- **Long-term data retention**: Many organizations require large-scale
+  data storage for extended periods, either for regulatory compliance
+  or historical data analysis. Cloud services provide an almost
+  limitless storage capacity, making it possible to keep data
+  accessible for as long as required at a reasonable cost. This is
+  where tiered storage becomes especially valuable.
+- **High-speed data ingestion**: Tiered storage can manage unpredictable or sudden data
+  influxes by supplementing local disks with cloud storage, ensuring optimal performance.
+- **Unlock unexplored opportunities:**Tiered storage in Aiven for Apache Kafka addresses
+  current storage challenges and enables new and innovative use cases that were
+  previously impractical or too expensive. By eliminating traditional storage
+  limitations, organizations gain the flexibility to support a wide range of
+  applications and workflows, including scenarios where using Apache Kafka was once
+  deemed impractical. This flexibility encourages creative thinking and redefines the
+  experience with Apache Kafka.
+
+## Pricing and charges
+
+Tiered storage costs depend on the amount of remote storage used, measured in GB/hour.
+Charges are based on the highest usage level within each hour.
+
+### Tiered storage charges
+
+Aiven charges for tiered storage data based on the data stored in remote tier, such as
+AWS S3. This means you pay for:
+
+- **Data stored in S3:** Billing is based on the highest amount of data stored in S3
+  within each hour, not the total amount stored over the hour.
+- **Local storage on the Aiven platform:** Local storage has a fixed cost based on the
+  provisioned capacity. You pay a set amount each month for a fixed amount of
+  local storage.
+
+Tiered Storage is designed to transfer **all** topic data to the remote tier, except
+for the active log segment where the latest data is being appended. This transfer occurs
+regardless of local storage retention settings.
+
+#### Example
+
+Imagine having a tiered topic with 40 TB of data and with tiered storage enabled.
+You set a local retention time of 10 TB. Consider the following:
+
+- You pay a fixed monthly cost for the local storage allocated to your service,
+  which includes the 10 TB retained locally.
+- You are charged for storing 40 TB of data in S3, even if some data is available locally
+  within the most recent 10 TB.This is because data is automatically transferred to S3,
+  and billing is based on the highest usage level within each hour.
```

Comment:
just to make it very clear, we could add a not on "approximately 40TB" and link the concepts section, wdyt? 
```suggestion
- You are charged for storing ~40 TB(*) of data in S3, even if some data is available locally
  within the most recent 10 TB.This is because data is automatically transferred to S3,
  and billing is based on the highest usage level within each hour.

NOTE: (*) approximately 40TB as it not considers the active segments that is only kept locally. For more info see link/concepts/ts/loca-remote-storage
```